lib/novu/http.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Novu.Http do
+  @moduledoc """
+  Helper functions for interacting with Req.
+  """
+  require Logger
+
+  @user_agent "Novu/" <> Mix.Project.config()[:version]
+
+  def delete(url) do
+    url
+    |> build_req()
+    |> Req.delete!()
+    |> handle_response()
+  rescue
+    e -> handle_response(e)
+  end
+
+  def post(url, body) do
+    url
+    |> build_req()
+    |> Req.post!(json: body)
+    |> handle_response()
+  rescue
+    e -> handle_response(e)
+  end
+
+  defp api_key, do: Application.fetch_env!(:novu, :api_key)
+
+  defp base_domain, do: Application.get_env(:novu, :domain, "")
```

Comment:
Unless I'm missing something we get an error for passing `""` as `base_url`:
```ex
iex(2)> Req.new(base_url: "") |> Req.get!()
** (ArgumentError) you must merge onto an absolute URI
    (elixir 1.13.3) lib/uri.ex:884: URI.merge/2
    (req 0.3.0) lib/req/steps.ex:39: Req.Steps.put_base_url/1
    (req 0.3.0) lib/req/request.ex:618: Req.Request.run_request/2
    (req 0.3.0) lib/req.ex:658: Req.request!/2
``` 

any particular reason to not require the domain with `fetch_env!` here?

Reply:
Thanks @samuelpordeus! That's on me, I have a commit locally that adds the default cloud URL here. What do you think about that vs fetch_env! ?

Reply:
`get_env` sounds awesome then! let's make sure to document that it points to the default cloud URL in case that env is missing 🎉